Add advanced usage of exception filters in readme

agrberg opened the pull request:
Callables can be passed as the values to handle errors differently
This is documented in the initializer at
https://github.com/rollbar/rollbar-gem/blob/d06b18d2a98d8effdde605355ebe14add7ae5d3d/lib/generators/rollbar/templates/initializer.rb#L41

brianr commented on README.md:
````diff
@@ -501,6 +501,16 @@ This behavior applies to uncaught exceptions, not direct calls to `Rollbar.error
 Rollbar.error(exception, :use_exception_level_filters => true)
 ```
 
+### Advanced Usage
````
"Advanced usage" is sorta vague - how about a heading that describes what this does, maybe "Dynamic levels" ?

brianr commented:
Thanks @agrberg!

@jondeandres can you review this to make sure this is accurate?

jondeandres commented:
@brianr sure, it's ok. I'll merge during the day. Thanks @agrberg!

agrberg commented:
Happy to help. This feature is far too useful to not be in the docs.

@brianr I was asleep when you commented so I did not have a chance to change it before it was merged. I think your wording is better so I've updated it and added the missing link to the TOC in https://github.com/rollbar/rollbar-gem/pull/481
